contrib/babelfishpg_tsql/src/collation.c

```diff
@@ -18,6 +18,7 @@
 #include "parser/parse_oper.h"
 #include "nodes/makefuncs.h"
 #include "nodes/nodes.h"
+#include "utils/varlena.h"
```

Comment:
.h file should be added following alphabet sequence, so this line should be near utils/xxx.h

Reply:
fixed